### ISSUE TITLE
NTR: Muscle biopsy: LAMA5 increased by immunostaining (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9419]
New term request Muscle biopsy: LAMA5 increased by immunostaining (elthat_181230144914):

**Comment by pnrobinson** [comment]
@MickeySegal
see #9418
Should we analogously use  laminin alpha-5 ?

**Comment by MickeySegal** [comment]
We were just following https://www.omim.org/entry/618138 "Muscle biopsy showed partial and variably decreased immunostaining for LAMA2 as well as increased immunostaining for LAMA5" but your usage seems more proper.  We will use the protein description in the title and include the gene symbol in the explanatory terms.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000659
- label: Elevated muscle fiber laminin alpha 5 expression
- parents: [Abnormal muscle fiber protein expression (HP:0030089)](https://hpo.jax.org/app/browse/term/HP:0030089)
- pmid: PMID:21953594
- def: A decreased amount of laminin alpha-5 in muscle tissue. This feature can be shown by immunohistochemistry or Western blotting of muscle tissue. Laminin subunit alpha-5 is a protein encoded by the LAMA5 gene.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9419
- synonyms:
